src/views/PeopleSearch/index.js

```diff
@@ -1130,90 +1154,74 @@ class PeopleSearch extends Component {
+                            this.state.majorSearchValue,
+                            this.state.minorSearchValue,
+                            this.state.hallSearchValue,
+                            this.state.classTypeSearchValue,
+                            this.state.homeCitySearchValue,
+                            this.state.stateSearchValue,
+                            this.state.countrySearchValue,
+                            this.state.departmentSearchValue,
+                            this.state.buildingSearchValue,
+                          );
+                        }}
+                        fullWidth
+                        variant="contained"
+                      >
+                        SEARCH
+                      </Button>
                     </Grid>
-                    <br />
-                    <Button
-                      color="primary"
-                      onClick={() => {
-                        this.search(
-                          this.state.includeAlumni,
-                          this.state.firstNameSearchValue,
-                          this.state.lastNameSearchValue,
-                          this.state.majorSearchValue,
-                          this.state.minorSearchValue,
-                          this.state.hallSearchValue,
-                          this.state.classTypeSearchValue,
-                          this.state.homeCitySearchValue,
-                          this.state.stateSearchValue,
-                          this.state.countrySearchValue,
-                          this.state.departmentSearchValue,
-                          this.state.buildingSearchValue,
-                        );
-                      }}
-                      fullWidth
-                      variant="contained"
-                    >
-                      SEARCH
-                    </Button>
-                  </Grid>
-                </CardActions>
-                <CardActions
-                  className={[classes.actions, 'card-expansion']}
-                  disableActionSpacing
-                  onClick={this.handleAdditionalOpsExpandClick}
-                  style={{
-                    marginTop: '-16px',
-                  }}
-                >
-                  <Grid container alignItems="baseline" justify="center">
-                    <Grid item>
-                      <IconButton
-                        className={classnames(classes.expand, {
-                          [classes.expandOpen]: this.state.additionalOpsExpanded,
-                        })}
-                        aria-expanded={this.state.additionalOpsExpanded}
-                        aria-label="Show more"
+                    {/* Clear All Button */}
+                    <Grid item xs={8} sm={'auto'}>
+                      <Button
+                        style={{backgroundColor: gordonColors.neutral.lightGray}}
+                        fullWidth
+                        disableElevation
```

Comment:
Is there a reason elevation is disabled here? To me at least, it makes it look less like a "clear" button and more like a disabled button that I can't interact with.